data/json/items/ranged.json

```diff
@@ -2117,11 +2117,12 @@
     "recoil": 30,
     "durability": 8,
     "burst": 5,
-    "clip_size": 20,
     "reload": 550,
```

Comment:
This is the time to actually insert the magazine. Default to 100 unless you have a reason to increase this (a magazine is particularly difficult to load)